### DISCUSSION TITLE
API reference?

### DISCUSSION THREAD

**Discussion by andypoly** [atteneder/glTFast#444] [Q&A] [answered]
The code seems fully commented but I did not see an API doc generated from it for reference (docs only skim the surface of import options) - is there one, could/should there be?
(I generated one with doxygen anyhow)

**Comment by atteneder** [comment, marked as answer]
Hi,

Good point. I've been preparing docs with doxygen locally for a while with the plans to eventually host them, but I haven't finished that.

I'm still mostly offline (parental leave) until end of August, so please be patient. As always: help is appreciated.

Thanks

**Reply by andypoly** [reply]
Oh no rush - was just wondering! Thanks for answer
